docker/docker_client_test.go

```diff
@@ -23,17 +23,17 @@ func TestDockerCertDir(t *testing.T) {
 	const nondefaultPerHostDir = "/this/is/not/the/default/certs.d"
 	const variableReference = "$HOME"
 	const rootPrefix = "/root/prefix"
-	const registryHostPort = "localhost:5000"
+	const registryHostPort = "thishostdefinitelydoesnotexist:5000"
 
-	systemPerHostResult := filepath.Join(systemPerHostCertDirPath, registryHostPort)
+	systemPerHostResultDocker := filepath.Join(systemPerHostCertDirPaths[1], registryHostPort)
```

Comment:
Maybe `systemPerHostCertDirPaths[len(systemPerHostCertDirPaths)-1]`, and rename the variable, to be both future-proof and not hard-code a magic `1` constant in here? (Very low-priority, the tests would reliably break if this ever got out of sync.)

Reply:
fixed